packages/skatejs/src/with-initializer.js

```diff
@@ -3,8 +3,9 @@
 export const withInitializer = (Base: Class<any> = HTMLElement): Class<any> =>
   class extends Base {
     static get observedAttributes() {
-      super.observedAttributes && super.observedAttributes();
+      let observed = super.observedAttributes;
```

Comment:
Something I just noticed: what happens when this is retrieved explicitly? i.e. not via `customElements.define()` - either before or after?